network-api/networkapi/wagtailpages/pagemodels/customblocks/articles.py

```diff
@@ -32,6 +32,13 @@ class Meta:
 class ArticleFullWidthImageBlock(blocks.StructBlock):
 
     image = ImageChooserBlock()
+    image_height = blocks.IntegerBlock(
+        default=410,
+        help_text='A custom height for this image. The image will be 1400px wide '
+                  'by this height. Note: This may cause images to look pixelated. '
```

Comment:
`image_height` is set to 410 by default, which was originally hard coded as `flll-1400x410`. We're keeping the 1400px but letting editors change the height. 

This notion enables the editors to set a height for an image they can't crop or edit on their own, and can reduce blurriness when certain image types are uploaded.